ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+***PCSX2 version:***
+``(For example, stable build v1.4.0 (or) development builds v1.5.xxx)``
+
+- **Note**: It's preferred to only report bugs which are present on the [latest development builds](http://pcsx2.net/download/development.html).
+
+
+***PCSX2 options:***
+``(For example, Clamping/Rounding, Speedhacks, GameFixes)``
+
+- **Note**: Generally it's recommended to have settings as default as possible and use ``Safest`` preset along with MTVU hack disabled. If there's no issue with these settings but there is an issue with some other non-default option, then please state it explicitly.
+
+
+***Plugins used:***
+``(Relevant/Non-default settings of GS/PAD/SPU2/CDVD/USB plugins)``
+
+
+***Description of the issue:***
+``(What's the problem?)``
+
+
+***How to reproduce the issue:***
+``(Detailed step by step guide on how to replicate the issue)``
+
+
+***Last known version to work:***
+``(If the issue was a regression it would be useful to mention the last known working version. However bisecting it down to the culprit build/commit is even better!)``
```

Comment:
Added it, Thanks :+1: 

FWIW, I added a comma after ``PCSX2 version`` and also changed ``it's`` **->** ``it would be``. Any opinion ? is it better (or) worse ?